tests/introspection/tests.py

```diff
@@ -118,18 +118,31 @@ def test_get_relations(self):
             relations = connection.introspection.get_relations(cursor, Article._meta.db_table)
 
         # That's {field_index: (field_index_other_table, other_table)}
```

Comment:
per the code updates, I think this should be `{field_name: (field_name_other_table, ...}`